src/ucp/api/ucp.h

```diff
@@ -1096,6 +1187,29 @@ unsigned ucp_worker_progress(ucp_worker_h worker);
 
 
 /**
+ * @ingroup UCP_WORKER
+ * @brief Poll stream oriented events on a specific worker.
+ *
+ * This non-blocking routine returns number of endpoints where requested events
+ * are raised. Endpoints will be filled at @a poll_eps array.
+ *
+ * @param [in]   worker    Worker to poll.
+ * @param [out]  poll_eps  Pointer to array of endpoints, should be
+ *                         allocated by user.
+ * @param [in]   max_eps   Number of endpoints fits to available memory
+ *                         space at @a poll_eps.
+ * @param [in]   flags     Modificators from UCS_STREAM_POLL_FLAG_* enum or 0.
```

Comment:
Reserved for future use (if we remove the flags), or add `@ref` to the relevant enum type